### DISCUSSION TITLE
ChatGPT-like interface

### DISCUSSION THREAD

**Discussion by CasKirch** [FlowiseAI/Flowise#1823] [Q&A] [answered]
Hi all,

Anyone know a good open-source/cheap ChatGPT UI to connect my Flowise bots to? I find this UI to be way better for internal agents than the chatbot widget. 

Casper

**Comment by Jaredude** [comment, marked as answer]
You can adapt one of the [Vercel AI template](https://vercel.com/templates?framework=next.js&type=ai) options.

The chatbot-ui project is nearly identical to ChatGPT. It's repo is here: https://github.com/mckaywrigley/chatbot-ui

You could fork and port those to support Flowise as the AI end point.

**Comment by CasKirch** [comment]
Great idea. Thanks HaredOp 1 mrt. 2024 21:48 schreef Jared Tracy ***@***.***>:
You can adapt one of the Vercel AI template options.
The chatbot-ui project is nearly identical to ChatGPT. It's repo is here: https://github.com/mckaywrigley/chatbot-ui
You could fork and port those to support Flowise as the AI end point.
